src/app/containers/ChartbeatAnalytics/canonical/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  chartbeatUID,
+  useCanonical,
+  getSylphidCookie,
+} from '../../../lib/analyticsUtils/chartbeat';
+
+const domain = 'test.bbc.co.uk';
+
+const inlinedScriptSnippet = () => `
```

Comment:
I know this is still in progress, but just wanted to add that this needs to be added to the `head` of the page. You can do this with `react-helmet`.